atomic_reactor/plugins/exit_store_metadata_in_osv3.py

```diff
@@ -207,7 +207,7 @@ def run(self):
             self.log.debug("annotations: %r", annotations)
             raise
 
-        labels = self.make_labels()
+        labels = self.make_labels(metadata)
```

Comment:
Technically `store_metadata` should know which labels are set already (not sure if reading `BUILD` env var is sufficient though). But I agree - lets not overcomplicate other plugins.

I'll rewrite the sendmail constructor to fetch task id and build id from two sources